### ISSUE TITLE
Command to update multiple button symbols

### ISSUE THREAD

**Issue by walmink** [fuggfuggfugg/sketch-dynamic-button-3.5#23]
The symbol support is great! Two things that could further improve efficiency:

1. Allow cmd+j to work on all symbols selected (currently, it only affects one)
2. Separate shortcut to update all symbols in the document (apart from making sure all your buttons are up-to-date without having to check individually, it's also a nice shortcut that means you don't have to select the button you've just changed the label of and what to update)

**Comment by fuggfuggfugg** [comment]
Can you elaborate more on point 2 with an example?

**Comment by JWhiteUX** [comment]
Agree with Fugg, can you elaborate on how you would use thaf feature? Why
would it improve your workflow?

**Comment by walmink** [comment]
Happy to elaborate!

Using the "global" shortcut to update all buttons would mean you can change a bunch of button labels and then update them all at once, without a need to select or check them individually (especially when combined with Auto-Layout plugin, the alignment could automatically update too, so you don't have to reposition it afterwards)

My second point of not having the select the button afterwards to do the update is irrelevant, as I just noticed you can do the cmd+j shortcut while the selection is on the input field of a symbol (used to update the button label).

**Comment by fuggfuggfugg** [comment]
The first issue is already fixed on the branch. Thanks for reporting. 

> Using the "global" shortcut to update all buttons would mean you can change a bunch of button labels and then update them all at once

Makes sense but currently there's no good way to identify if a component is a dynamic button or not. Cannot rely on the name of the layers since users can change.  Once I have a solution, then it opens up door to other possibilities eg: Run dynamic button on text change. 

Also haven't played with Auto-Layout plugin, will check it out.

**Comment by fuggfuggfugg** [comment]
Can I close this for now?

**Comment by walmink** [comment]
Yup, if you don't have a sure way of identifying buttons, then it's not possible for now. I'll have a look if there is a plugin to select one symbol and then through a shortcut select all the same symbols!

**Comment by fuggfuggfugg** [comment]
@WWalmink 

Actually I may have a way to tag a component once a plugin is run the first time. The subsequent times you run a plugin, I may be able to detect based on the tag.

**Comment by walmink** [comment]
Hm, that's pretty cool, but also a little hard to predict, as the user cannot see which buttons have or haven't been triggered before. Limiting it to symbols (of which there typically are only a few) might be a solution? So any instance of a symbol tagged as a button will update with the global shortcut. Just an idea.

**Comment by fuggfuggfugg** [comment]
Is your request similar to [this](https://github.com/fuggfuggfugg/sketch-dynamic-button-3.5/issues/19#issuecomment-284743011)
